app/controllers/tenants/projects_controller.rb

```diff
@@ -64,7 +74,7 @@ def new_categories
   def new_params
     new_param = project_params
     new_param[:categories] = new_categories
-    new_param[:tenant_id] = current_user.tenant.id
+    new_param[:tenant_id] = @project.tenant.id
```

Comment:
This is the line that is causing the error. NoMethodError - undefined method `tenant' for nil:NilClass:

There is a current_user, but not an @project apparently